pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -278,6 +283,14 @@ func (o *StartBuildOptions) Complete(f *clientcmd.Factory, in io.Reader, out, er
 	}
 	o.EnvVar = env
 
+	// Handle Docker build arguments. In order to leverage existing logic, we
+	// first create an EnvVar array, then convert it to []docker.BuildArg
+	buildArgs, _, err := cmdutil.ParseEnv(o.Args, in)
+	if err != nil {
+		return fmt.Errorf(strings.Replace(err.Error(), "environment variable", "build arg", -1))
```

Comment:
this is fragile.  better to just change ParseEnv to be more generic in its error messages.